lib/SILOptimizer/Mandatory/TFCanonicalizeCFG.cpp

```diff
@@ -403,14 +423,14 @@ void SingleExitLoopTransformer::initialize() {
 
 llvm::DenseMap<SILValue, SILValue>
 SingleExitLoopTransformer::computeEscapingValues() const {
```

Comment:
can this be a static method?

Reply:
Unfortunately no as it relies on some object state.